packages/server/src/definitions/automations.ts

```diff
@@ -15,6 +15,7 @@ export interface TriggerOutput {
 
 export interface AutomationContext extends AutomationResults {
   steps: any[]
+  stepsById?: Record<string, any>
```

Comment:
How come this is optional? Would it not be safer to default it to `{}`? All of the code I can see referencing this does so with an `!` so it feels a bit pointless to have it listed here as optional.